plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1408,6 +1408,7 @@ def build_transaction(  # pylint: disable=too-many-arguments
         tx_params = {
             "nonce": nonce,
             "value": tx_args["value"] if "value" in tx_args else 0,
+            "gas": 1,  # set this as a placeholder to avoid estimation on buildTransaction()
```

Comment:
we should only set this placeholder if there's no manually provided `gas` (see below)

Reply:
it will be overwritten if it is present, and we avoid a check